src/extensions/default/AutoUpdate/main.js

```diff
@@ -321,15 +322,14 @@ define(function (require, exports, module) {
      */
     function setupAutoUpdate() {
         updateJsonHandler = new StateHandler(updateJsonPath);
+        updateDomain.on('data', receiveMessageFromNode);
 
         updateDomain.exec('initNode', {
             messageIds: MessageIds,
             updateDir: updateDir,
             requester: domainID
         });
 
-        updateDomain.on('data', receiveMessageFromNode);
-        initState();
```

Comment:
`initState()` not required anymore?

Reply:
Now initState() is called from nodeDomainInitialized() function. so it is not required  here.